### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by ntzm**

### CODE REVIEW

File: src/Fixer/ControlStructure/NoBreakCommentFixer.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ private function isNoBreakComment(Token $token)
 
         $text = preg_quote($this->configuration['comment_text'], '~');
 
-        return Preg::match("~^((//|#)\\s*${text}\\s*)|(/\\*\\*?\\s*${text}\\s*\\*/)$~i", $token->getContent());
+        return (bool) Preg::match("~^((//|#)\\s*${text}\\s*)|(/\\*\\*?\\s*${text}\\s*\\*/)$~i", $token->getContent());
```
Comment (keradus) [inline src/Fixer/ControlStructure/NoBreakCommentFixer.php]:
```diff
-(bool)
+1 ===
```

**Comment by keradus** [comment]
my diff patch (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3779/files/3af46c57d08ec227bdca9eb4428b869d7ed9ee9a#diff-d2157600e5393b9d87c2cd0a61273255) was wrongly applied. I let myself fix it

**Comment by ntzm** [comment]
Oops was in a rush 😄

**Comment by keradus** [comment]
Thank you @ntzm.

**Pull request merged**